packages/react-form-state/src/FormState.ts

```diff
@@ -71,27 +73,52 @@ export default class FormState<
   static Nested = Nested;
 
   static getDerivedStateFromProps<T>(newProps: Props<T>, oldState: State<T>) {
-    const {initialValues, onInitialValuesChange} = newProps;
+    const {
+      initialValues,
+      onInitialValuesChange,
+      externalErrors = [],
+    } = newProps;
+
+    let nextState;
```

Comment:
I thought using let is discouraged instead you should have separate return statements.

Reply:
Good call. I've cleaned this up.